src/ontology/mondo-edit.obo

```diff
@@ -282636,7 +282635,7 @@ xref: OMIM:612851 {source="MONDO:equivalentTo"}
 xref: Orphanet:2073 {source="OMIM:612851"}
 xref: UMLS:C2748508 {source="OMIM:612851", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline"}
 is_a: MONDO:0100049 {source="DC-OMIM:612851"} ! narcolepsy, susceptibility to
-relationship: predisposes_towards MONDO:0021107 {source="OMIM:612851"} ! narcolepsy
+relationship: predisposes_towards MONDO:0021107 {source="OMIM:612851"} ! hereditary narcolepsy
```

Comment:
same as above. should be "predisposes towards" narcolepsy (and not hereditary narcolepsy)